SonarQube.TeamBuild.PreProcessor/Interfaces/IAnalyzerProvider.cs

```diff
@@ -7,6 +7,7 @@
 
 using SonarQube.Common;
 using SonarQube.TeamBuild.Integration;
+using SonarQube.TeamBuild.PreProcessor.Roslyn;
```

Comment:
Following up on the unit test failure - now resolved. See [this comment](https://github.com/SonarSource-VisualStudio/sonar-msbuild-runner/pull/190/files#r51862621).